### PR TITLE
Add layout_stop-watch

### PR DESCRIPTION
**Pull request by YegorKochetkov**
- [DEMO LINK](https://YegorKochetkov.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    z-index: 1;
+
+    @include circle(20px);
+
+    transform: translate(-50%, -25%);
+
+    background-color: $minuteColor;
+  }
+
+  &__minute {
```
Comment (Yurii837) [inline src/styles/main.scss:61]: Ето же секунды

File: src/styles/main.scss [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    height: $minuteArrowWidth;
+
+    animation: clock 60s steps(60) infinite;
+    transform-origin: 0 #{$minuteArrowWidth / 2};
+
+    background-color: $minuteColor;
+    border-radius: $minuteArrowWidth;
+  }
+
+  &__hour {
```
Comment (Yurii837) [inline src/styles/main.scss:74]: Соотвецтвенно минуты

File: src/styles/main.scss [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  }
+
+  &__minute {
+    @extend %arrowsPosition;
+
+    width: $minuteArrowLength;
+    height: $minuteArrowWidth;
+
+    animation: clock 60s steps(60) infinite;
+    transform-origin: 0 #{$minuteArrowWidth / 2};
```
Comment (Yurii837) [inline src/styles/main.scss:68]: можно просто centr bottom

File: src/styles/main.scss [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+@keyframes clock {
+  from {
+    transform: rotate(-90deg);
+  }
+
+  to {
+    transform: rotate(270deg);
+  }
+}
```
Comment (Yurii837) [inline src/styles/main.scss:96]: Я б не задавал %arrowsPosition rotate(-90deg) соотвецтвенно в @keyframes clock { from 0 to 360}

**Review by Yurii837 — APPROVED** [review]

File: src/styles/main.scss [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+    border-radius: $secondArrowWidth;
+  }
+
+  &__minute {
+    @extend %arrowsPosition;
+
+    height: $minuteArrowWidth;
+    width: $minuteArrowLength;
+
+    animation: clock 60s steps(60) infinite;
```
Comment (illiarerun) [inline src/styles/main.scss:80]: your clock doesn't work properly, check it for yourself& a minute clock should make one rotation in 1 hour, not a minute^ and also it should move smoothly

File: src/styles/main.scss [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+
+    background-color: $secondColor;
+    border-radius: $secondArrowWidth;
+  }
+
+  &__minute {
+    @extend %arrowsPosition;
+
+    height: $minuteArrowWidth;
+    width: $minuteArrowLength;
```
Comment (illiarerun) [inline src/styles/main.scss:78]: swap these values and then you can use normal keyframes

File: src/styles/main.scss [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+@keyframes clock {
+  from {
+    transform: translate(0, -50%) rotate(-90deg);
+  }
+
+  to {
+    transform: translate(0, -50%) rotate(270deg);
+  }
```
Comment (illiarerun) [inline src/styles/main.scss:94]: like that
```suggestion
  from {
    transform: translate(0, -50%) rotate(0deg);
  }

  to {
    transform: translate(0, -50%) rotate(360deg);
  }
```

File: src/styles/main.scss [hunk 8]
```diff
@@ -0,0 +1,95 @@
+// extends
+%arrowsPosition {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+
+  transform-origin: left center;
```
Comment (illiarerun) [inline src/styles/main.scss:7]: and also change it to bottom center

**Review by illiarerun — CHANGES_REQUESTED** [review]
Good job, let's wor on fixes now:

**Review by illiarerun — APPROVED** [review]

**Pull request closed without merging**